SkietbaanBE/SkietbaanBE/Controllers/FeaturesController.cs

```diff
@@ -32,5 +32,23 @@ public User Search(string username)
             }
             return null;
         }
+
+        //// PUT: api/User/Update
+        [HttpPut]
+        [ActionName("Update")]
+        public async Task<IActionResult> PutUserMember(String username, [FromBody] User user)
+        {
+            User dbUser = null; //assume user does not exist
+            dbUser = _context.Users.Where(u => u.Username == user.Username)
+                .FirstOrDefault<User>();
+
+            //now updating user details
+            dbUser.MemberID = user.MemberID;
```

Comment:
You will have null reference exceptions here if the dbUser was not found

Reply:
Adjusted